plugin/src/test/java/io/jenkins/plugins/forensics/git/miner/GitMinerStepITest.java

```diff
@@ -43,6 +45,34 @@ public void shouldFillTableDynamically() {
                 .hasFileName("file").hasAuthorsSize(1).hasCommitsSize(1);
     }
 
+    /** Verifies that the mining process is incremental. */
+    @Test
+    public void shouldMineRepositoryIncrementally() throws IOException {
+        writeFileAsAuthorFoo("First");
+
+        FreeStyleProject job = createJobWithMiner();
+        Run<?, ?> build = buildSuccessfully(job);
+
+        getJenkins().assertLogContains("created report for 2 files", build);
+
+        build = buildSuccessfully(job);
+
+        getJenkins().assertLogContains("created report for 0 files", build);
+    }
+
+    /** Verifies that the latest revision id is saved in the build result. */
+    @Test
+    public void shouldSaveLatestRevisionId() {
+        FreeStyleProject job = createJobWithMiner();
+        Run<?, ?> build = buildSuccessfully(job);
+
+        BuildData buildData = (BuildData)build.getAllActions().get(0);
+        String latestId = buildData.lastBuild.revision.getSha1().getName();
```

Comment:
Wouldn't it be simpler to use the actual repository? I.e., `getHead()` in `GitITest`?